From 3281ff68b723b5d669374f78f45ffc3878b8ca5f Mon Sep 17 00:00:00 2001
From: Alfredo Moralejo <amoralej@redhat.com>
Date: Wed, 6 Apr 2016 20:29:25 +0200
Subject: [PATCH] Packstack should not specify release when installint rdo repo

When installing rdo repo file, packstack looks for
rdo-release-%(version)s-%(release)s.rpm but only rdo-release-%(version)s
should be used as it always points to the current release of the
package.

Change-Id: Ibf3ee875fdb4d58be3a424f6d28fc0cdfb517171
Closes-Bug:  1567021
(cherry picked from commit 8debfd463915678da875512c7cd283c629fbbea8)
---
 packstack/plugins/prescript_000.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/packstack/plugins/prescript_000.py b/packstack/plugins/prescript_000.py
index 5165f4a..6f52759 100644
--- a/packstack/plugins/prescript_000.py
+++ b/packstack/plugins/prescript_000.py
@@ -1215,7 +1215,7 @@ def manage_rdo(host, config):
     match = re.match(r'^(?P<version>\w+)\-(?P<release>\d+\.[\d\w]+)\n', out)
     version, release = match.group('version'), match.group('release')
     rdo_url = ("https://www.rdoproject.org/repos/openstack-%(version)s/"
-               "rdo-release-%(version)s-%(release)s.rpm" % locals())
+               "rdo-release-%(version)s.rpm" % locals())
 
     server = utils.ScriptRunner(host)
     server.append("(rpm -q 'rdo-release-%(version)s' ||"
